contrib/inventory/ec2.py

```diff
@@ -286,6 +286,11 @@ def read_settings(self):
         if config.has_option('ec2', 'elasticache'):
             self.elasticache_enabled = config.getboolean('ec2', 'elasticache')
 
+        # Combine with Autoscaling groups tags?
+        self.autoscaling_enabled = True
```

Comment:
should this be False by default (it's False by default in the ini file, but if someone uses an old ini file with the updated script, odd things might happen)